src/lib/yang/alarms.lua

```diff
@@ -84,6 +109,20 @@ end
 
 -- Contains a table with all the declared alarms.
 local alarm_list = {}
+function alarm_list:lookup (key)
+   return alarm_list[key]
```

Comment:
Since `lookup` is also a member of `alarm_list`, perhaps you would prefer to separate things a bit; store the list in a member of the `alarm_list` table?